src/madpack/madpack.py

```diff
@@ -565,13 +559,13 @@ def _db_upgrade(schema, filehandle, db_madlib_ver):
     sc = uu.ScriptCleaner(schema, portid, con_args, ch)
     info_(this, "Script Cleaner initialized ...", False)
 
-    ch.drop_changed_uda()
+    function_drop_str = get_madlib_function_drop_str(schema)
```

Comment:
Is there a specific reason for creating SQL queries and then running them, instead of calling the original functions (`drop_changed_uda()` etc) ?

Reply:
Those are for dropping the functions, etc. from changelist. Now we use a different query to get all madlib functions.